Code/GraphMol/molopstest.cpp

```diff
@@ -8051,6 +8051,38 @@ void testRingFamilies() {
 void testRingFamilies() {}
 #endif
 
+void testSetTerminalAtomCoords() {
```

Comment:
What happens when you add two terminal atoms to the same atom?

Reply:
Should be fine. The AddHs() code uses that all the time.